### PR TITLE
Serverless flows in ODC

### PR DESCRIPTION
**Pull request by Preeticp**
This PR:

-  Adds a create a serverless app assembly with a reused module for creating serverless app using dev perspective. 
- Adds a splitting traffic between revisions assembly with a new module for splitting traffic using dev perspective
- Adds commented out includes for upcoming CLI sections in the above assemblies.
- This is relevant to the 4.3 Branch.
- This has been reviewed by the SMEs and the QE.
- This has been discussed with Ashleigh Brenan, the Serverless writer.

### CODE REVIEW

**Comment by abrennan89** [comment]
LGTM, thanks for adding this @Preeticp :slightly_smiling_face:

File: serverless/creating-serverless-applications.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+// include::modules/serverless-creating-serverless-applications-using-cli.adoc[leveloffset=+1]
+
+.Prerequisites
+To create serverless applications using the *Developer* perspective ensure that:
+
+* You have xref:../web-console/web-console.adoc#web-console[logged in to the web console].
+* You are in the xref:../web-console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
+* You have the appropriate xref:../authentication/using-rbac.adoc#default-roles_using-rbac[roles and permissions] in a project to create applications and other workloads in {product-title}.
+* You have xref:../serverless/installing-openshift-serverless.html#installing-serverless-operator_installing-openshift-serverless[installed the Openshift Serverless Operator].
```
Comment (ahardin-rh) [inline serverless/creating-serverless-applications.adoc]: replace .html with .adoc in the last few links

**Comment by ahardin-rh** [comment]
The build error indicates that a reference to serverless/traffic-splitting-between-revisions.adoc is not recognized

**Comment by Preeticp** [comment]
@ahardin-rh I have fixed the build errors, the preview currently shows me a 4.2 version of the import from git to create app flow, probably cause the app flow PR is not merged yet. The build error turned out to be on account of some commented out includes. I replaced the 'include' with 'to do' and it seems fine now. Also fixed the html/adoc and the relative links. Thanks for the guidance!

**Comment by ahardin-rh** [comment]
Great! Thanks, Preeti!

**Pull request merged**